Cargo.toml

```diff
@@ -45,3 +45,11 @@ rand = "0.8.5"
 
 [lib]
 crate-type = ["cdylib", "rlib"]
+
+[[example]]
+name = "cct"
+required-features = ["cct"]
+
+[[example]]
+name = "cri"
+required-features = ["cri"]
```

Comment:
This tells cargo that these examples require the following features. This makes cargo skip the examples when the feature is disabled, or give a more appropriate error if someone is trying to explicitly build this example without the required features.

Instead of:
```
error[E0412]: cannot find type `CRI` in this scope
  --> examples/cri.rs:16:19
   |
16 |         let cri : CRI = spc.as_ref().try_into()?;
   |                   ^^^ not found in this scope
```
We get:
```
error: target `cri` in package `colorimetry` requires the features: `cri`
Consider enabling them by passing, e.g., `--features="cri"`
```